cellfinder_napari/detect.py

```diff
@@ -30,6 +30,8 @@
 
 
 def detect() -> FunctionGui:
+    progress_bar = ProgressBar(min=0, max=1)
```

Comment:
are you sure this is right? The default `step` value is `1` so I don't think this is going to work https://napari.org/magicgui/usage/_autosummary/magicgui.widgets.ProgressBar.html

Reply:
I've removed these arguments since it's possible to create a `ProgressBar` without arguments.